src/api/app/components/workflow_run_filter_checkbox_component.rb

```diff
@@ -0,0 +1,28 @@
+class WorkflowRunFilterCheckboxComponent < ApplicationComponent
+  def initialize(text:, filter_item:, selected_filter:, amount:, icon: '')
+    super
+
+    @text = text
+    @sanitized_key = text.parameterize.underscore
+    @filter_item = filter_item
+    @selected_filter = selected_filter
+    @amount = amount || 0
+    @icon = icon
+  end
+
+  def icon_tag
+    tag.i(class: ['me-1', @icon]) if @icon != ''
+  end
+
+  private
+
+  def workflow_run_filter_matches?
```

Comment:
Hm, why is this a component?

Reply:
Instead of a partial?

Reply:
Definitely a partial. So the other filters actually.

Reply:
They were components, they still are. Refactoring to be tackled later. A card exists already for that